release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -116,6 +116,11 @@ With this release, there are several updates to the *Developer* perspective of t
 ** Cluster roles accessible to a project
 ** Actions visible on the *+Add* page
 ** Item types in the *Developer Catalog*
+* See the common updates to the *Pipeline details* and *PipelineRun details* page visualization performing the following actions:
```

Comment:
Pipeline details repeated twice.

Reply:
@DebarghoGhosh The first instance is for **Pipeline** details and the second instance is for **PipelineRun** details page. So, we don't have repetition here.